test/e2e/data/infrastructure-docker/main/cluster-template-cross-ns-topology/cluster.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: '${CLUSTER_NAME}'
+spec:
+  topology:
+    classNamespace: '${CLUSTER_CLASS_NAMESPACE}'
```

Comment:
Wondering if we can avoid creating a new template by changing cluster-with-topology.yaml so it uses empty string as a default value in case CLUSTER_CLASS_NAMESPACE is not set